pkg/api/v1/register.go

```diff
@@ -23,8 +23,11 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
+// GroupName is the group name use in this package
+const GroupName = api.GroupName
```

Comment:
> Yes-- if we had an import tree test, this would have broken it :)
> pkg/apis/x/v1 may not import pkg/apis/x.

In fairness, this breaks it right here: https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/conversion.go#L23 .  We seem to be rather far from where you want to be.

Will update.